### PR TITLE
UCT: Check that length for get_zcopy is larger than min_zcopy.

### PR DESCRIPTION
**Pull request by yosefe**

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test FAILed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1256/ for details.

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/3203/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1267/ for details.

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/3214/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1268/ for details.

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1269/ for details.

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/3215/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/3216/ for details (Mellanox internal link).

**Comment by yosefe** [comment]
@shssf pls review

**Comment by shssf** [comment]
:+1: 
Other variants failed in coverity test with "This greater-than-or-equal-to-zero comparison of an unsigned value is always true."

File: src/uct/ib/rc/base/rc_ep.h [hunk 1]
```diff
@@ -10,6 +10,7 @@
 #include "rc_iface.h"
 
 #include <uct/api/uct.h>
+#include <ucs/debug/debug.h>
```
Comment (shamisp) [inline src/uct/ib/rc/base/rc_ep.h]: why debug.h added ? If the new macro depends on debug probably the include should go to uct_iface.h

Comment (yosefe) [inline src/uct/ib/rc/base/rc_ep.h]: leftovers - will remove

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1300/ for details.

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/3243/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/3246/ for details (Mellanox internal link).

**Comment by yosefe** [comment]
@shamisp fixed

**Comment by shamisp** [comment]
@MattBBaker corresponding ugni code has to be updates

**Comment by MattBBaker** [comment]
@yosefe I've made a PR  against your branch with the UGNI updates.

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1334/ for details.

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/3281/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/3283/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/3284/ for details (Mellanox internal link).

**Pull request merged**